spec/robots/accession/publish_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+require File.expand_path(File.dirname(__FILE__) + '/../../../robots/accession/publish')
+
+describe Robots::DorRepo::Accession::Publish do
+  let(:druid) { 'druid:oo000oo0001' }
+
+  it 'includes behavior from LyberCore::Robot' do
+    robot = Robots::DorRepo::Accession::Shelve.new
+    expect(robot.methods).to include(:work)
+  end
+
+  it 'calls .publish_metadata if that method is defined (e.g. item)' do
+    object = double(:publish_metadata => true)
+    expect(object.respond_to?(:publish_metadata)).to be true
```

Comment:
yeah, this was helpful for debugging, but realize its not an actual test, removing all of them